server/schedulers/hot_test.go

```diff
@@ -46,6 +47,10 @@ func (s *testHotSchedulerSuite) TestGCPendingOpInfos(c *C) {
 	opt := mockoption.NewScheduleOptions()
 	newTestReplication(opt, 3, "zone", "host")
 	tc := mockcluster.NewCluster(opt)
+	for id := uint64(1); id <= 10; id++ {
```

Comment:
why add it?

Reply:
The refactored `operator.Builder` will check the validity of the target leader, and it is illegal if the store id of the target leader does not exist.